src/io/flutter/logging/FlutterLogView.java

```diff
@@ -283,6 +295,56 @@ public void setSelected(AnActionEvent e, boolean state) {
     }
   }
 
+  private class ChannelPanel extends JPanel {
+    class LoggerCheckBox extends JBCheckBox implements ActionListener {
+      @NotNull
+      private final LoggingChannel channel;
+
+      LoggerCheckBox(@NotNull LoggingChannel channel) {
+        super(channel.name);
+        this.channel = channel;
+        setToolTipText(channel.description);
+        setSelected(channel.enabled);
+        addActionListener(this);
+      }
+
+      @Override
+      public void actionPerformed(ActionEvent e) {
+        app.getFlutterLog().enable(channel, isSelected());
+      }
+    }
+
+    ChannelPanel(List<LoggingChannel> channels) {
+      setLayout(new GridLayout(0, 1));
+      channels.forEach(c -> add(new LoggerCheckBox(c)));
+    }
+  }
+
+  private class ConfigureChannelsAction extends AnAction {
+    ConfigureChannelsAction() {
+      super("Configure channels...");
+    }
+
+    @Override
+    public void actionPerformed(AnActionEvent e) {
+      app.getFlutterLog().getLoggingChannels().thenAccept(channels -> ApplicationManager.getApplication().invokeAndWait(() -> {
+        final ChannelPanel panel = new ChannelPanel(channels);
+        final Rectangle visibleRect = logTree.getVisibleRect();
+        final Point topRight = new Point(logTree.getLocationOnScreen().x + visibleRect.width - 150 /* TODO: make dynamic */,
```

Comment:
nit: make this TODO a // and maybe indicate how dynamic it should be.

Reply:
Done!